pkg/wipeout/wipeout.go

```diff
@@ -28,95 +28,44 @@ import (
 func Wipeout(ctx context.Context, client connection.RegistryClient, projectID string, jobs int) {
 	log.Debugf(ctx, "Deleting everything in project %s", projectID)
 	project := "projects/" + projectID + "/locations/global"
-	// Wipeout resources in groups to ensure that children are deleted before parents.
-	{
-		log.Debugf(ctx, "Deleting artifacts")
-		taskQueue, wait := core.WorkerPool(ctx, jobs)
-		wipeoutArtifacts(ctx, client, taskQueue, project+"/apis/-/versions/-/specs/-")
-		wipeoutArtifacts(ctx, client, taskQueue, project+"/apis/-/versions/-")
-		wipeoutArtifacts(ctx, client, taskQueue, project+"/apis/-/deployments/-")
-		wipeoutArtifacts(ctx, client, taskQueue, project+"/apis/-")
-		wipeoutArtifacts(ctx, client, taskQueue, project)
```

Comment:
This last line (39) is deleting top-level artifacts, which won't be deleted by deleting APIs in cascading mode (because top-level artifacts aren't under APIs).

Reply:
Ah. I'll need to update the test. It didn't catch that.